src/main/java/com/sublinks/sublinksapi/person/entities/LinkPersonPerson.java

```diff
@@ -0,0 +1,102 @@
+package com.sublinks.sublinksapi.person.entities;
+
+import com.sublinks.sublinksapi.person.enums.LinkPersonPersonType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.util.Date;
+import java.util.Objects;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.SourceType;
+import org.hibernate.proxy.HibernateProxy;
+
+/**
+ * The Post class represents a post in a community or instance. It contains various attributes such
```

Comment:
This description is wrong. Looks like copy-paste from Post class?

Reply:
Ohh yeah sry